assets/src/edit-story/app/media/utils/useTranscodeVideo.js

```diff
@@ -28,8 +28,37 @@ import { useCurrentUser } from '../../currentUser';
 
 const isDevelopment = process.env.NODE_ENV === 'development';
 
+/**
+ * Returns file basename without extension.
+ *
+ * @param {File} file File object.
+ * @param {string} file.name File name.
+ * @return {string} File name without extension.
+ */
 const getFileName = ({ name }) => name.split('.').slice(0, -1).join('.');
 
+/**
+ * Determines whether it's a video file.
+ *
+ * @param {File} file File object.
+ * @param {string} file.type File type.
+ * @return {boolean} Whether it's a video file.
+ */
+const isVideo = ({ type }) => type.startsWith('video/');
+
+const GB_IN_BYTES = 1024 * 1024 * 1024;
+
+/**
+ * Checks whether the file size is too large for transcoding.
+ *
+ * @see https://github.com/ffmpegwasm/ffmpeg.wasm/tree/9b56b7f05b552c404aa0f62f46bed2592d9daf06#what-is-the-maximum-size-of-input-file
+ *
+ * @param {File} file File object.
+ * @param {number} file.size File size.
+ * @return {boolean} Whether the file is too  large.
+ */
+const isFileTooLarge = ({ size }) => size >= 2 * GB_IN_BYTES;
```

Comment:
Wondering if use [bytesToMB](https://github.com/google/web-stories-wp/blob/f445eacfe8eabf2a0eb107a5dbc466245677b156/assets/src/edit-story/app/uploader/useUploader.js#L56). 
```
const isFileTooLarge = ({ size }) => bytesToMB( size ) >= 2000;
```

Reply:
`bytestoMB` rounds numbers. It's OK for some better human readable number, but for doing actual comparison it's better to use unrounded numbers. Don't want false results here due to rounding.